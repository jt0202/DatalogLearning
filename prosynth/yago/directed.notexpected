nicole_kassell	im_not_single
king_vidor	el_abuelo
saeed_akhtar_mirza	colonel_chabert_1994_film
trevor_nunn	shelter_dogs
marty_feldman	bintou
