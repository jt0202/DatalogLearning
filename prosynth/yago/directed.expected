denis_héroux	naked_massacre
john_lennon	imagine_film
sunil_dutt	reshma_aur_shera
jacques_doillon	the_hussy
tzang_merwyn_tong	a_wicked_tale
alex_chandon	night_pastor
becky_stark	califunya
françois_dupeyron	the_officers_ward_film
bob_saget	becoming_dick
abel_gance	la_digue_film
saeed_akhtar_mirza	mohan_joshi_hazir_ho
al_pacino	chinese_coffee
bert_haanstra	fanfare_film
pieter_verhoeff	mates_film
king_vidor	our_daily_bread_1934_film
leonid_trauberg	the_return_of_maxim
román_viñoly_barreto	la_bestia_debe_morir
jeanpierre_mocky	un_drôle_de_paroissien
emilio_estevez	bobby_2006_film
norman_mailer	maidstone_film
emilio_fernández	siempre_tuya
king_vidor	better_times_1919_film
mai_zetterling	doctor_glas_film
shuji_kataoka	sm_hunter
dane_cook	tourgasm
kenneth_branagh	frankenstein_1994_film
márton_keleti	two_confessions
warren_beatty	dick_tracy_1990_film
jill_hennessy	the_acting_class
burt_reynolds	the_man_from_left_field
witold_leszczyński	matthews_days
norodom_sihanouk	an_ambition_reduced_to_ashes
vic_morrow	a_man_called_sledge
chris_rock	i_think_i_love_my_wife
leonid_trauberg	alone_1931_film
alan_alda	the_four_seasons_film
luis_bayón_herrera	jettatore
ishirō_honda	godzilla_1954_film
palito_ortega	amigos_para_la_aventura
marco_ferreri	i_love_you_1986_film
marco_ferreri	the_conjugal_bed
isabel_coixet	map_of_the_sounds_of_tokyo
bert_haanstra	mirror_of_holland
eric_byler	americanese
norodom_sihanouk	my_village_at_sunset
tony_bui	three_seasons
warren_beatty	heaven_can_wait_1978_film
abel_gance	les_gaz_mortels
alan_rudolph	afterglow_film
david_mamet	house_of_games
alan_rudolph	love_at_large
kenneth_branagh	the_magic_flute_2006_film
maurice_cloche	born_of_unknown_father
saeed_akhtar_mirza	arvind_desai_ki_ajeeb_dastaan
denis_leary	favorite_deadly_sins
kenneth_branagh	hamlet_1996_film
tage_danielsson	the_adventures_of_picasso
noboru_iguchi	robogeisha
mai_zetterling	loving_couples_1964_film
sunil_dutt	daaku_aur_jawan
abel_gance	the_mask_of_horror
quentin_tarantino	grave_danger
lou_diamond_phillips	dangerous_touch
rick_rubin	tougher_than_leather_film
abel_gance	the_zone_of_death
jamil_dehlavi	the_blood_of_hussain
guan_hu	dirt_1994_film
vilsoni_hereniko	the_land_has_eyes
tom_hanks	that_thing_you_do
abel_gance	poliche
ishirō_honda	destroy_all_monsters
alex_chandon	drillbit_film
andrew_lane	jake_speed
abel_gance	la_folie_du_docteur_tube
marco_ferreri	dont_touch_the_white_woman
ward_kimball	man_in_space
palito_ortega	el_tío_disparate
michel_ocelot	princes_et_princesses
burt_reynolds	the_end_film
nelson_george	life_support_film
henrigeorges_clouzot	manon_film
michael_dowse	fubar_2_film
abel_gance	un_drame_au_château_dacre
michel_houellebecq	la_possibilité_dune_île_film
king_vidor	love_never_dies_1921_film
saeed_akhtar_mirza	albert_pinto_ko_gussa_kyon_aata_hai
iain_softley	backbeat_film
henrigeorges_clouzot	le_corbeau
alan_rudolph	trouble_in_mind_film
marco_ferreri	bye_bye_monkey
sunil_dutt	yaadein_1964_film
ltÉric_Rochantgt	the_patriots_film
lillete_dubey	lt30_Days_in_September_Playgt
brian_dannelly	saved
jorge_lanata	deuda
alan_rudolph	welcome_to_la
alan_alda	war_of_nerves_mash
marco_ferreri	break_up_film
alan_rudolph	choose_me
lou_diamond_phillips	love_takes_wing
jeanpierre_bekolo	les_saignantes
lucas_belvaux	la_raison_du_plus_faible
john_wayne	the_green_berets_film
kamal_haasan	virumaandi
marco_ferreri	the_story_of_piera
clive_barker	hellraiser
kevin_munroe	teenage_mutant_ninja_turtles_film_series
emilio_fernández	flor_silvestre
abel_gance	le_périscope
michael_dowse	kids_in_america_2011_film
lotte_reiniger	the_adventures_of_prince_achmed
frank_d_gilroy	desperate_characters
denis_héroux	valérie
sandra_goldbacher	me_without_you_film
abel_gance	end_of_the_world_1931_film
saeed_akhtar_mirza	salim_langde_pe_mat_ro
craig_ferguson	ill_be_there_film
max_jacoby	dust_2009_film
henrigeorges_clouzot	la_vérité_film
raymond_bernard	les_misérables_1934_film
jacques_doillon	the_pirate_1984_film
emilio_fernández	maclovia_1948_film
joe_mcstravick	murphy_771
michael_keaton	the_merry_gentleman
nicolas_roeg	aria_1987_film
isabel_coixet	the_secret_life_of_words
param_gill	hotel_hollywood
joss_whedon	goners
henrigeorges_clouzot	the_wages_of_fear
guy_ritchie	the_hire
emilio_fernández	río_escondido_1948_film
isabel_coixet	my_life_without_me
abel_gance	the_ironmaster
paul_auster	lulu_on_the_bridge
bill_condon	dreamgirls_film
michael_dowse	its_all_gone_pete_tong
abel_gance	au_secours
abel_gance	le_droit_à_la_vie
abel_gance	the_torture_of_silence
steven_e_de_souza	possessed_2000_film
pierre_andre	im_not_single
emilio_fernández	la_choca
michael_tolkin	the_rapture_film
abel_gance	louise_film
abel_gance	the_tenth_symphony
king_vidor	buds_recruit
lucas_belvaux	rapt_2009_film
maija_digiorgio	bitter_jester
cliff_robertson	the_pilot_film
nasser_taghvai	captain_khorshid
marco_ferreri	liza_film
tonia_marketaki	crystal_nights
al_pacino	looking_for_richard
clive_barker	lord_of_illusions
dani_levy	alles_auf_zucker
king_vidor	the_turn_in_the_road
king_vidor	poor_relations
king_vidor	wild_oranges
leonid_trauberg	the_youth_of_maxim
jaromil_jireš	the_cry_1963_film
françois_dupeyron	monsieur_ibrahim
marshall_brickman	the_manhattan_project_film
alex_chandon	bad_karma_1991_film
yves_angelo	colonel_chabert_1994_film
emilio_fernández	the_torch_film
benito_alazraki	roots_film
alan_alda	sweet_liberty
jaime_camino	spain_again
marco_ferreri	the_ape_woman
carlos_atanes	proxima
frank_buck_animal_collector	jungle_cavalcade_1941_film
jacques_feyder	latlantide_1921_film
marco_ferreri	ludienza
marty_feldman	in_god_we_trut
cliff_robertson	j_w_coop
michael_dowse	fubar_film
yılmaz_güney	yol
jacques_doillon	family_life_1985_film
steven_e_de_souza	arnolds_wrecking_co
king_vidor	three_wise_fools_1923_film
noboru_iguchi	the_machine_girl
norodom_sihanouk	the_last_days_of_colonel_savath
john_wayne	the_undefeated_1969_film
paul_auster	blue_in_the_face
neil_gaiman	a_short_film_about_john_bolton
fernando_cerchio	lulu_1953_film
david_e_durston	i_drink_your_blood
nasser_taghvai	tales_of_kish
